### DISCUSSION TITLE
ECT simulation using simnibs

### DISCUSSION THREAD

**Discussion by Freeman0922** [simnibs/simnibs#394] [2. General]
Dear experts.

We are attempting to perform the e-field simulation when applying ECT to subjects with MDD using simNIBS. Is it possible to simulate the ECT simulation using the simNIBS? And if so, is the setting up method similar to that of the tDCS setting? Any comment is appreciated. Thank you.  
Best  
Tatsu

**Comment by bjornsig** [comment]
Yes, this is fine as a first approximation. For a more detailed discussion see:

https://www.nature.com/articles/s41380-024-02567-9

Which also mentions upcoming more refined approaches. 

Best regards,
Bjorn
